shared/lgcrt/include/lgcrt/LgcRtDialect.td

```diff
@@ -313,6 +312,13 @@ def TraceRayOp : LgcRtOp<"trace.ray", [Memory<[(read)]>, WillReturn]> {
   }];
 }
 
+// =========================================================================================================
+def TraceRayOp : LgcRtOp<"trace.ray", [Memory<[(read)]>, WillReturn]> {
```

Comment:
Why the inheritance?

Reply:
This is due to some upcoming changes that are internal-only for now.